### DISCUSSION TITLE
Splitting window and seeing other variables/variants

### DISCUSSION THREAD

**Discussion by AxisMeetsWorld** [mrdbourke/pytorch-deep-learning#504] [Ideas]
Hi all,
       I am used to working with Visual Studio, in which you can split your screen or open up a couple of tabs so you can look at different classes of your code and adeptly view different aspects of your code. 👍  I was wondering if there's an easy way to do that in Google Colab, and [this article](https://algo-ai.work/en-US/blog/2023/01/21/post-1430-2-2/) helped figure that out! 👍 
       The only caveat I might mention is it seems to be a little outdated when it comes to the vertical view. You need to select the ellipsis and then select "two rows" and you can see 2 different panes. 
       ❓ I did wonder if anyone else might have any suggestions regarding viewing variables (sort of like a watch window) ❓ Would love to know of any other split screen/cross referencing tips anyone else might know other than using the **mirror cell in tab** trick?

**Comment by mrdbourke** [comment]
Hi @AxisMeetsWorld ,

Good question! 

I actually haven't seen this before on Google Colab (so I just learned something too!).

In terms of seeing variables, have you tried the "Variables" tab on the left hand side of the Google Colab window? (I'm using Google Colab Pro so this may only be a feature there 🤔)

See the attached image for reference: 

<img width="1624" alt="Screenshot 2023-06-29 at 3 08 28 pm" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/16750345/d486211a-6d51-4322-a144-4c1e767f83fa">

**Reply by AxisMeetsWorld** [reply]
That answered my question; thank you @mrdbourke.
